mmv1/templates/terraform/pre_create/integrations_client.go.erb

```diff
@@ -0,0 +1,5 @@
+// Translate `createSampleIntegrations` to `createSampleWorkflows`
```

Comment:
I see here that this is ensuring that the new field name that's used in the provider is 'translated back' to match the old field name that's still used in the API. Is the plan that the API will be changed soon, and this code will be removed at the same time?

Reply:
The change in the API is not planned since we do not expect users to consume our api's directly. However, due to legacy field name, this is being referred as workflow instead integration in the api. Since terraform would allow users to directly use this field, we intend to fix the field name for terraform and later would plan to fix this field in api as well. We would remove this code once we mark the field in the api as deprecated.